ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.14__4.7.yaml

```diff
@@ -0,0 +1,226 @@
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.16.0_knative-serving-src:
+    name: knative-v0.16.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.17.1_knative-eventing-contrib-src:
+    name: knative-v0.17.1
+    namespace: openshift
+    tag: knative-eventing-contrib-src
+  openshift_knative-v0.17.2_knative-eventing-src:
+    name: knative-v0.17.2
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.17.3_knative-serving-src:
+    name: knative-v0.17.3
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.18.0_knative-eventing-contrib-src:
+    name: knative-v0.18.0
+    namespace: openshift
+    tag: knative-eventing-contrib-src
+  openshift_knative-v0.18.2_knative-serving-src:
+    name: knative-v0.18.2
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.18.4_knative-eventing-src:
+    name: knative-v0.18.4
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.18.6_knative-eventing-src:
+    name: knative-v0.18.6
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v0.19.0_knative-serving-src:
+    name: knative-v0.19.0
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_knative-v0.19.1_knative-eventing-kafka-src:
+    name: knative-v0.19.1
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v0.19.2_knative-eventing-src:
+    name: knative-v0.19.2
+    namespace: openshift
+    tag: knative-eventing-src
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v0.19.0_knative-serving-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.19.0:knative-serving-src
+    openshift_knative-v0.19.1_knative-eventing-kafka-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.19.1:knative-eventing-kafka-src
+    openshift_knative-v0.19.2_knative-eventing-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-src
+  to: serverless-operator-src
+- context_dir: .
+  dockerfile_path: serving/ingress/Dockerfile
+  from: base
+  to: knative-openshift-ingress
+- context_dir: .
+  dockerfile_path: knative-operator/Dockerfile
+  from: base
+  to: knative-operator
+- context_dir: .
+  dockerfile_path: openshift-knative-operator/Dockerfile
+  from: base
+  to: openshift-knative-operator
+- context_dir: olm-catalog/serverless-operator
+  from: base
+  to: serverless-bundle
+promotion:
+  disabled: true
+  name: openshift-serverless-v1.14.0
+  namespace: openshift
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.7"
```

Comment:
The above should be changed to:
```
releases:
  latest:
    release:
      channel: stable
      version: "4.7"
```
The stable version is out. We don't need to test with a nightly build.